app/views/api/v2/hosts/show.json.rabl

```diff
@@ -3,7 +3,7 @@ object @host
 extends "api/v2/hosts/main"
 
 child :interfaces => :interfaces do
-  extends "api/v2/interfaces/base"
+  extends "api/v2/interfaces/main"
```

Comment:
Here 'api/v2/interfaces/base' is a subset of 'api/v2/interfaces/main' w.r.t interface attributes.